docs/installation.md

```diff
@@ -88,69 +101,43 @@ And the other one is responsible for persisting the **read model** aka **project
 
 ### Step 3 - Configuration
 
-For email sending (mandatory):
+#### 3.1 Email sending (mandatory):
 
 Copy `config/autoload/mail.local.php.dist` to `config/autoload/mail.local.php` and make your adjustments.
 
-For read model (mandatory):
+#### 3.2 Read model (mandatory):
 
-Copy `config/autoload/doctrine.local.php.dist` to `config/autoload/doctrine.local.php` and make your adjustments.
-
-For MySQL Event Store (mandatory: choose MySQL or Postgres):
+ - Copy `config/autoload/doctrine.local.php.dist` to `config/autoload/doctrine.local.php` and make your adjustments.
+ - Execute `CREATE DATABASE todo;` on your MySQL instance.
 
-Copy `config/autoload/mysql_event_store.local.php.dist` to `config/autoload/mysql_event_store.local.php` and make your adjustments.
+#### 3.3 Event Store
 
-For Postgres Event Store (mandatory: choose MySQL or Postgres):
+ - Copy `config/autoload/mysql_event_store.local.php.dist` to `config/autoload/mysql_event_store.local.php` and make your adjustments.
+ - Execute the scripts located at:
+   - `vendor/prooph/pdo-event-store/scripts/mysql/01_event_streams_table.sql`
+   - `vendor/prooph/pdo-event-store/scripts/mysql/02_projections_table.sql`
+ - Create empty stream: Run `php scripts/create_event_stream.php`
 
-Copy `config/autoload/postgres_event_store.local.php.dist` to `config/autoload/postgres_event_store.local.php` and make your adjustments.
-
-For PDO Snapshot Store (optional - it uses the same PDO connection as for given EventStore)
+#### 3.4 PDO Snapshot Store (optional)
+*PDO Snapshot Store uses the same PDO connection as for given EventStore*
 
 Copy `config/autoload/pdo_snapshot_store.local.php.dist` to `config/autoload/pdo_snapshot_store.local.php` and make your adjustments.
 
-For MongoDB Snapshot Store (optional)
```

Comment:
Good point. I am still trying to write this without the current "Cyclomatic Komplexitity" 
and less decisions to be made but also I want to communicate all options available.